concepts/05 Widgets/FileManager/30 Validation/20 Upload Settings.md

```diff
@@ -0,0 +1,130 @@
+The widget allows you to configure upload settings: 
+
+- [maxFileSize](/api-reference/10%20UI%20Widgets/dxFileManager/1%20Configuration/upload/maxFileSize.md '/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/upload/#maxFileSize') option - Specifies the maximum upload file size. The widget does not upload a file and displays the following error message when the file's size exceeds the maximum upload file size.
```

Comment:
With this formating, all the text will be displayed on a single line. I suggest formatting the list as shown, for example, in [this file](https://raw.githubusercontent.com/DevExpress/devextreme-documentation/20_1/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/sorting/mode.md).

Reply:
ok